File: FrontEnd/src/app/features/pickupPoints/classes/dtos/pickupPoint-read-dto.ts
import { CoachRouteDropdownVM } from '../../../coachRoutes/classes/view-models/coachRoute-dropdown-vm'

export interface PickupPointReadDto {

    id: number
    description: string
    coachRoute: CoachRouteDropdownVM
    exactPoint: string
    time: string
    coordinates: string
    isActive: boolean

}

File: API/Features/PickupPoints/Dtos/Form/PickupPointReadDto.cs
namespace API.Features.PickupPoints {

    public class PickupPointReadDto {

        public int Id { get; set; }
        public string Description { get; set; }
        public string ExactPoint { get; set; }
        public string Time { get; set; }
        public string Coordinates { get; set; }
        public bool IsActive { get; set; }

        public CoachRouteDropdownVM CoachRoute { get; set; }

    }

}